src/http_utils.cpp

```diff
@@ -193,6 +193,8 @@ const char* http_utils::http_post_encoding_multipart_formdata = MHD_HTTP_POST_EN
 const char* http_utils::application_octet_stream = "application/octet-stream";
 const char* http_utils::text_plain = "text/plain";
 
+const char* http_utils::upload_filename_template = "libhttpserver.XXXXXX";
```

Comment:
Maybe `static const char*` because not used outside of this source file?

Reply:
Changed this to
`static const std::string upload_filename_template = "libhttpserver.XXXXXX";`
and removed the declaration from http_utils.hpp